Cargo.toml

```diff
@@ -23,6 +23,9 @@ default = []
 builder = ["derive_builder"]
 unstable = []
 
+[package.metadata.docs.rs]
+all-features = true
```

Comment:
Actually isnt `rustdoc-args = ["--cfg", "docsrs"]` also required? Or does docs.rs set the cfg automatically these days?